ospfd/ospf_te.c

```diff
@@ -3390,16 +3482,21 @@ static uint16_t show_vty_link_subtlv_unrsv_bw(struct vty *vty,
 }
 
 static uint16_t show_vty_link_subtlv_rsc_clsclr(struct vty *vty,
-						struct tlv_header *tlvh)
+						struct tlv_header *tlvh,
+						json_object *json)
 {
 	struct te_link_subtlv_rsc_clsclr *top;
 
 	check_tlv_size(TE_LINK_SUBTLV_DEF_SIZE, "Resource class/color");
 
 	top = (struct te_link_subtlv_rsc_clsclr *)tlvh;
 	if (vty != NULL)
-		vty_out(vty, "  Resource class/color: 0x%x\n",
-			(uint32_t)ntohl(top->value));
+		if (!json)
+			vty_out(vty, "  Resource class/color: 0x%x\n",
+				(uint32_t)ntohl(top->value));
+		else
+			json_object_string_addf(json, "resClassColor", "0x%x",
```

Comment:
`resourceClassColor` maybe? Don't see a reason to shorten here.

Reply:
Done